spec/src/main/asciidoc/architecture.adoc

```diff
@@ -32,6 +32,39 @@ This specification documents the component specification contents of each releas
 
 The specifications for each release are listed in reverse chronological order.
 
+[[microprofile7.0]]
+=== MicroProfile 7.0
+Based on MicroProfile's time-boxed release process, this is a major release that does include backward incompatible changes. This release no longer packages Jakarta EE 10 Core Profile, instead it just specifies the dependency on Jakarta EE 10 Core Profile. This release does not include MicroProfile Metrics as MicroProfile Telemetry 2.0 offers the Metrics capabilities in addition to Logs and Tracing. Therefore, MicroProfile Metrics became a standalone specification.
```

Comment:
Some recommendations:

"Based on MicroProfile's time-boxed release process, this is a major release that does include backward incompatible changes. This release no longer packages Jakarta EE 10 Core Profile, instead, it specifies the dependency on Jakarta EE 10 Core Profile. This release replaces MicroProfile Metrics with MicroProfile Telemetry 2.0 which offers the Metrics capabilities in addition to Logs and Tracing."

Removes "just". Simplifies wording on Metrics being replaces. I think we should not state that Metrics is a standalone spec within this spec.